### PR TITLE
[Rules] Add custom error responses (beta)

### PR DESCRIPTION
**Pull request by pedrosousa**
Addresses PCX-3699.

### CODE REVIEW

File: content/rules/custom-error-responses/error-tokens.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: Error tokens
+pcx-content-type: reference
+weight: 4
+meta:
+  title: Custom error response error tokens
```
Comment (marciocloudflare) [inline content/rules/custom-error-responses/error-tokens.md]: This sentence sounds a bit convoluted. `Custom error response tokens`? `Custom error response with error tokens`?

File: content/rules/custom-error-responses/parameters.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+title: Parameters
+pcx-content-type: reference
+weight: 3
+meta:
+  title: Custom error responses parameter reference
```
Comment (marciocloudflare) [inline content/rules/custom-error-responses/parameters.md]: Not sure if I understood this sentence. Should this be `Custom error responses' parameter reference`? So the parameter reference for custom error responses?

Comment (pedrosousa) [reply content/rules/custom-error-responses/parameters.md]: Yes, that's it. I'm adding more context to the page title to improve search, but the sentences do look a bit long.

**Review by marciocloudflare — APPROVED** [review]

File: content/rules/custom-error-responses/error-tokens.md (hunk 1 above)
Comment (pedrosousa) [reply content/rules/custom-error-responses/error-tokens.md]: I'll update to your first proposal. Thanks!

Comment (pedrosousa) [reply content/rules/custom-error-responses/error-tokens.md]:
```suggestion
  title: Custom error response tokens
```

File: content/rules/custom-error-responses/parameters.md (hunk 2 above)
Comment (pedrosousa) [reply content/rules/custom-error-responses/parameters.md]:
```suggestion
  title: Custom error response parameters
```

**Pull request merged**